gpio/ds18x20_sensor.py

```diff
@@ -83,10 +84,10 @@ def __init__(self, publishers, dev_cfg):
         if self.smoothing:
             self.temp_readings = [None] * 5
 
-        #configure_output for homie etc. after debug output, so self.comm is clean
+        # Configure_output for homie etc. after debug output, so self.comm is clean
         configure_device_channel(self.comm, is_output=True, output_name=OUT_TEMP,
                                  datatype=ChanType.FLOAT, name="temperatur reading",
-                                 unit='°C' if self.temp_unit=='C' else '°F')
+                                 unit="C" + self.temp_unit)
```

Comment:
The only question I have is this one change. The new line changes what at lines does. 

I don't have a ds18x20 sensor and I didn't write the Homie stuff so I'm not 100% on what it means but I believe this controls the unit the value is published with when using Homie. The docs for `configure_devioce_channel1 show it using °C or °F for temperature in the list of units. And the line that populates `self.temp_unit` appears to be using, expecting `C` or `F`. 

So as I read it the old line would use `°C` if `self.temp_unit` is `C` and `°F` otherwise.

The new line would generate `CC` or `CF` as the unit, which doesn't make sense to me. What am I missing?

Reply:
You are missing nothing, this is a very stupid typo. I removed the wrong character.

I'll correct that.